src/components/OptionList/OptionList.tsx

```diff
@@ -109,11 +117,6 @@ export function OptionList({
 
   const optionsMarkup = optionsExist
     ? normalizedOptions.map(({title, options}, sectionIndex) => {
```

Comment:
You've removed the title from the `normalizedOptions` - so this destructuing looks like it will always reference an undefined value.  Is `title` ever going to be set here or are you always overwriting the `title` that got set on from the props line 63?